tern/analyze/common.py

```diff
@@ -222,6 +228,68 @@ def convert_to_pkg_dicts(pkg_dict):
     return pkg_list
 
 
+def get_deb_package_license(deb_copyright):
+    '''
+    Given the debian copyright texts,
+    1. parse each copyright text
+    2. filter out all the available licenses
+    3. returns a list of unique licenses found inside
+    the copyright text
+    '''
+    collected_paragraphs = list()
+    pkg_licenses = set()
+    for paragraph in iter(debcon.get_paragraphs_data(deb_copyright)):
+        if 'format' in paragraph or 'format-specification' in paragraph:
+            cp = CopyrightHeaderParagraph.from_dict(paragraph)
+        elif 'files' in paragraph:
+            cp = CopyrightFilesParagraph.from_dict(paragraph)
+        elif 'license' in paragraph:
+            cp = CopyrightLicenseParagraph.from_dict(paragraph)
+        else:
+            cp = CatchAllParagraph.from_dict(paragraph)
+        collected_paragraphs.append(cp)
+
+    deb_pkg_data = DebianCopyright(collected_paragraphs).to_dict()
+    for paragraph in deb_pkg_data.get("paragraphs"):
+        pkg_license = paragraph.get("license") \
+            if 'license' in paragraph else ''
```

Comment:
Because we seem to be doing the same check here, regardless of which paragraphs we get.

Reply:
Done!